shared-helpers/src/views/sign-in/FormSignIn.tsx

```diff
@@ -1,27 +1,24 @@
-import React, { useContext } from "react"
+import React from "react"
 import type { UseFormMethods } from "react-hook-form"
-import { Field, Form, NavigationContext, t } from "@bloom-housing/ui-components"
+import { useRouter } from "next/router"
+import { t } from "@bloom-housing/ui-components"
 import { Button, Heading } from "@bloom-housing/ui-seeds"
 import { CardSection } from "@bloom-housing/ui-seeds/src/blocks/Card"
 import { FormSignInErrorBox } from "./FormSignInErrorBox"
 import { NetworkStatus } from "../../auth/catchNetworkError"
 import { BloomCard } from "../components/BloomCard"
-import { useRouter } from "next/router"
 import { getListingRedirectUrl } from "../../utilities/getListingRedirectUrl"
 import styles from "./FormSignIn.module.scss"
 
 export type FormSignInProps = {
   control: FormSignInControl
-  onSubmit: (data: FormSignInValues) => void
   networkStatus: NetworkStatus
   showRegisterBtn?: boolean
+  children: React.ReactNode
 }
 
 export type FormSignInControl = {
   errors: UseFormMethods["errors"]
-  handleSubmit: UseFormMethods["handleSubmit"]
-  register: UseFormMethods["register"]
-  watch: UseFormMethods["watch"]
```

Comment:
Why was watch not being used and now being removed?

Reply:
It was likely just missed at some point, because of the way we were typing this, VSCode / linting didn't alert us to an unused item on the type.